node_modules/oae-core/topnavigation/topnavigation.html

```diff
@@ -155,7 +155,7 @@ <h1 id="topnavigation-target" class="sr-only">__MSG__SKIP_MENU_TARGET__</h1>
                 <i class="fa fa-home"><span class="sr-only">__MSG__HOME__</span></i>
             </a>
         </li>
-        <li>
+        <li id="menuitem-notifications">
```

Comment:
Needs to be namespaced with `topnavigation-`. Something like `topnavigation-notifications-container` would be consistent with the codebase.